edx_proctoring/backends/software_secure.py

```diff
@@ -390,3 +392,30 @@ def get_video_review_aes_key(self):
         Returns the aes key used to encrypt the video review url
         """
         return self.video_review_aes_key
+
+    def get_instructor_url(
+        self, course_id, user, exam_id=None, attempt_id=None,
+        show_configuration_dashboard=False, encrypted_video_review_url=None
+    ):
+        """
+        Returns the url for video reviews
+        """
+        # video_review_url is required for PSI backend
+        if not encrypted_video_review_url:
+            return None
+
+        try:
+            aes_key = codecs.decode(self.video_review_aes_key, "hex")
+            decrypted_video_url = decode_and_decrypt(encrypted_video_review_url, aes_key).decode("utf-8")
+        except Exception as err:  # pylint: disable=broad-except
+            log.exception(
+                'Could not decrypt video url for attempt_id=%(attempt_id)s '
+                'due to the following error: %(error_string)s',
+                {'attempt_id': attempt_id, 'error_string': str(err)}
+            )
+            return None
+
+        # reformat video url as per MST-871 findings
+        reformatted_url = decrypted_video_url.replace('DirectLink-Generic', 'DirectLink-HTML5')
```

Comment:
Shouldn't this line is also in the try catch block?

Reply:
Yes, thanks for catching that